### PR TITLE
This is POM Changes by Sachin

### PR DESCRIPTION
**Pull request by rahulstar7**
V1:B-155544545/AddingDEpendenciesInPOM

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -38,6 +38,12 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/jstl/jstl -->
```
Comment (Dalchandsahu) [inline pom.xml:41]: resolve confict

Comment (ManmohanKaushik) [inline pom.xml:41]: please formated the dependency using java formatted

Comment (rahulstar7) [reply pom.xml:41]: Yes I resolved the conflicts

Comment (rahulstar7) [reply pom.xml:41]: I did the formatting

**Comment by ManmohanKaushik** [comment]
good

**Pull request merged**